### PR TITLE
Board meeting notes May 2021

### PR DESCRIPTION
**Pull request by lenucksi**

### CODE REVIEW

File: content/about/board/2020-05-26.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+* Cedric Williams (Treasurer)
+
+### Directors Absent
+
+* None
+
+
+### Members Present
+* Sebastian Spier
+
```
Comment (dicortazar) [inline content/about/board/2020-05-26.md:33]: Clare Dillon was present as well

**Review by dicortazar — CHANGES_REQUESTED** [review]

File: content/about/board/2020-05-26.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+
+### Directors Present
+
+* Jacob Green 
+* Danese Cooper (Chair)
+* Isabell Drost-Fromm (President)
+* Georg Grütter (Vice President)
+* Daniel Izquierdo
+* Russel Rutledge (Secretary)
```
Comment (rrrutledge) [inline content/about/board/2020-05-26.md]:
```suggestion
* Russell Rutledge (Secretary)
```

File: content/about/board/2020-05-26.md (hunk 1 above)
Comment (rrrutledge) [reply content/about/board/2020-05-26.md:33]:
```suggestion
Clare Dillon
```

**Review by rrrutledge — CHANGES_REQUESTED** [review]
Two name changes.

File: content/about/board/2020-05-26.md (hunk 1 above)
Comment (lenucksi) [reply content/about/board/2020-05-26.md:33]: True indeed, thanks for pointing out!

**Comment by lenucksi** [comment]
Addressed both change requests.

File: content/about/board/2020-05-26.md [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+- Adjourned 10:00 CEST
+
+## Role Call
+
+
+### Directors Present
+
+* Jacob Green 
+* Danese Cooper (Chair)
+* Isabell Drost-Fromm (President)
```
Comment (MaineC) [inline content/about/board/2020-05-26.md]:
```suggestion
* Isabel Drost-Fromm (President)
```

**Comment by voborgus** [comment]
What is the process of review?

**Review by MaineC — APPROVED** [review]

**Review by dicortazar — APPROVED** [review]

**Comment by MaineC** [comment]
@lenucksi can this be merged?

**Comment by lenucksi** [comment]
> @lenucksi can this be merged?

I see nothing against it. Didn't merge for @dicortazar's red review yet, although all changes have been addressed. Feel free to merge yourself too.

**Pull request merged**